File: python/8kyu/no_zeros_for_heros.py
def no_boring_zeros(n):
  if len(str(n)) == 1:
    return n
  else:
    n = list(str(n))[::-1]

    if n[-1] == '-':
      neg_sign = n.pop(-1)
    else:
      neg_sign = ''
    
    n = int(''.join(n))
    n = list(str(n))[::-1]


    z = n.insert(0, neg_sign)

    z = int(''.join(n))
    print(z)



n = -1050   
no_boring_zeros(n)   

File: javascript/8_kyu/no_zeros_for_heros.js
// No zeros for heros
// https://www.codewars.com/kata/570a6a46455d08ff8d001002/train/javascript

function noBoringZeros(n) {
    var n = n.toString();
    var lis_n = n.split('');
  
    if (n.length == 1) {
      return parseInt(n);
    }

    for (var i = lis_n.length - 1; i >= 0; i--) {
      if (lis_n[i] == 0) {
        lis_n.pop();
      } else {
        return parseInt(lis_n.join(''));
      }
    }
    return parseInt(lis_n.join(''));
  }